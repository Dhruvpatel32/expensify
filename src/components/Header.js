import {NavLink} from 'react-router-dom';
import React from 'react';
const Header =()=>(<div>
    <h1>Expensify</h1>
 <NavLink to="/" activeClassName="is-active" exact={true}>home</NavLink>
 <NavLink to="/create" activeClassName="is-active">add</NavLink>
 <NavLink to="/editexpanse" activeClassName="is-active">Edit</NavLink>

</div>
);
export default Header;