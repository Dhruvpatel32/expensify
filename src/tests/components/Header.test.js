import React from 'react';
import {shallow} from 'enzyme';
import Header from '../../components/Header';
import Adapter from 'enzyme-adapter-react-16';
test('should render Header',()=>{
   const wrapper=shallow(<Header/>)
  expect(wrapper).toMatchSnapshot();
})