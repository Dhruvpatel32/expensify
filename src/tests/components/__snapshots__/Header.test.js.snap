// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    add
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/editexpanse"
  >
    Edit
  </NavLink>
</div>
`;
